### DISCUSSION TITLE
how to write log into file

### DISCUSSION THREAD

**Discussion by minhanghuang** [abseil/abseil-cpp#1398] [Q&A] [answered]
Can abseil write the log to the specified file like glog?

**Comment by derekmauro** [comment, marked as answer]
Abseil currently has no builtin ability to write to a file. Instead, we provide an interface, [`absl::LogSink`](https://abseil.io/docs/cpp/guides/logging#LogSink) to allow you to do whatever you need to do with logs, including writing to a file.

A future Abseil release may include `absl::LogSink`s that write to a file so that you don't have to implement them yourself.

**Reply by minhanghuang** [reply]
> Abseil currently has no builtin ability to write to a file. Instead, we provide an interface, [`absl::LogSink`](https://abseil.io/docs/cpp/guides/logging#LogSink) to allow you to do whatever you need to do with logs, including writing to a file.
> 
> A future Abseil release may include `absl::LogSink`s that write to a file so that you don't have to implement them yourself.

Thank you.
